controllers/rule_controller.go

```diff
@@ -57,3 +107,47 @@ func (r *RuleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&apiv1.ConfigMap{}).
 		Complete(r)
 }
+
+func (r *RuleReconciler) updateRulesConfigmap(ctx context.Context, data string) error {
+
+	var oathkeeperRulesConfigmap apiv1.ConfigMap
+
+	if err := r.Get(ctx, r.RuleConfigmap, &oathkeeperRulesConfigmap); err != nil {
+		if apierrs.IsNotFound(err) {
+
+			oathkeeperRulesConfigmap = apiv1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      r.RuleConfigmap.Name,
+					Namespace: r.RuleConfigmap.Namespace,
+				},
+				Data: map[string]string{"rules": data},
+			}
+
+			err := retry.Do(func() error {
```

Comment:
A question: How many times does it retry, and what is the retry timing policy (fixed intervals, exponential backoff, etc) ?